.github/scripts/run_ui_tests.sh

```diff
@@ -28,4 +28,10 @@ while netstat -lnt | awk '$4 ~ /:8080$/ {exit 1}'; do
     else 
         sleep 10
     fi
-done
\ No newline at end of file
+done
+
+IN="$INPUT"
+for i in $(echo $IN | tr ";" "\n"); do
```

Comment:
I'd suggest using a different separator like comma instead of semicolons, since semicolons are a common terminator used in shell scripts:

```shell
cd dir1; ls -la
```